thinc/backends/cpu_kernels.hh

```diff
@@ -143,9 +141,7 @@ void cpu_reduce_mean(A* means__bo, const A* X__to, const L* lengths__b,
     static_assert(std::is_integral<L>::value, "Array length should be integral");
 
     for (const L* length = lengths__b; length < lengths__b + B; ++length) {
-        if (*length == 0)
-            continue;
-        else if (*length < 0)
+        if (*length < 0)
```

Comment:
Not sure if it makes sense, but it could also have a `length == 0` and then it just returns `0` for clarity?

Reply:
So, I think you are proposing:

```
else if (length == 0) {
    means__bo += O;
    continue;
}
```

?

Reply:
yes